src/main/java/yevhen/bookstore/model/Book.java

```diff
@@ -0,0 +1,86 @@
+package yevhen.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+
+@Entity
```

Comment:
u can add @Data ann. (lombok), and remove setters and getters, because lombok generates it automatically